packages/lambdas/src/shared/bundle-to-html.ts

```diff
@@ -252,17 +256,28 @@ function extractFhirTypesFromBundle(bundle: Bundle): {
         procedures.push(resource as Procedure);
       } else if (resource?.resourceType === "Observation") {
         const observation = resource as Observation;
+        const isVitalSigns = observation.extension?.find(
+          ext => ext.valueCodeableConcept?.coding?.[0]?.code === "CCD Vitals"
+        );
+        const isSocialHistory = observation.extension?.find(
+          ext => ext.valueCodeableConcept?.coding?.[0]?.code === "CCD Social History"
+        );
+        const isLaboratory = observation.category?.find(
+          category => category.text === "laboratory"
```

Comment:
would recommend doing these comparisons like so `ext => ext.valueCodeableConcept?.coding?.[0]?.code.toLower() === "ccd vitals"` - just to account for any possible caps issues